config.toml

```
@@ -120,30 +120,30 @@ With Flux, everything is controlled through pull requests, which means no learni
 [params.resources]
 title   = "Resources"
 summary = """
-Pellentesque id metus et quam sodales efficitur. Suspendisse pretium lacus at arcu ornare tristique. Fusce porttitor sagittis finibus.
+If you are new to Flux, you might want to check out some of the following resources to get started.
 """
 
 ## Resource items
 [[params.resources.items]]
-title       = "Quisque aliquet libero ut ultrice"
+title       = "Rawkode Live: Introduction to GitOps Toolkit with Stefan Prodan"
 description = """
-Integer tortor ex, porttitor quis porttitor at, mattis et erat. Fusce in ullamcorper elit.
+Check out Stefan's introduction to Flux at Rawkode Live.
 """
-url         = "#"
+url         = "https://www.youtube.com/watch?v=HqTzuOBP0eY"
 
 [[params.resources.items]]
-title       = "Quisque aliquet libero ut ultrice"
+title       = "The Power of GitOps with Flux v2 & GitOps Toolkit with Leigh Capili"
 description = """
-Integer tortor ex, porttitor quis porttitor at, mattis et erat. Fusce in ullamcorper elit.
+Leigh shows off the Power of GitOps with Flux. This is part 1 - stay tuned for the next installment.
 """
-url         = "#"
+url         = "https://www.youtube.com/watch?v=0v5bjysXTL8"
 
 [[params.resources.items]]
-title       = "Quisque aliquet libero ut ultrice"
+title       = "Flux Deep Dive: A GitOps Approach to Progressive Delivery - Stefan Prodan & Hidde Beydals"
 description = """
-Integer tortor ex, porttitor quis porttitor at, mattis et erat. Fusce in ullamcorper elit.
+In their KubeCon talk, Hidde and Stefan, both maintainers of Flux, show a GitOps approach to Progressive Delivery.
 """
-url         = "#"
+url         = "https://www.youtube.com/watch?v=8v94nUkXsxU"
```

```suggestion
title       = " Introduction To GitOps and GitOps Toolkit"
description = """
<some description>
"""
url         = "https://www.youtube.com/watch?v=qQBtSkgl7tI"
```

Looking back, this is a better introduction than the deep dive we did on KubeCon where we tried to include too much information in 40 minutes.